Wikipedia/Code/NSURL+WMFLinkParsing.m

```diff
@@ -180,7 +180,13 @@ + (NSURL *)wmf_desktopURLForURL:(NSURL *)url {
 #pragma mark - Properties
 
 - (BOOL)wmf_isWikiResource {
-    return [self.path wmf_isWikiResource];
```

Comment:
wmf_isWikiResource on a string should only be used for paths. If anything, we can remove that public interface and only allow checks on a URL